res/values-pt-rPT/strings.xml

```diff
@@ -1276,22 +1276,22 @@
     <string name="The_winner_will_receive_x_plasma_">O ganhador receberá %s plasma!</string>
     <string name="admin_commands">"
 Comandos:
-!kick [0–31]
-!reset
-!welcome {mensagem}
+!kick [0–31] - Expulsar jogador
+!reset - Reiniciar o jogo
+!welcome {Mensagens de boas-vindas}
```

Comment:
Why plural in "mensagens"?

Reply:
Porque se refere a várias mensagens, não apenas uma. Se estivesse a falar de uma única mensagem, diríamos "mensagem de boas-vindas". Portanto, "mensagens de boas-vindas" indica múltiplas saudações.

Reply:
Mas se você quiser, corrijo isso ainda hoje.

Reply:
> Porque se refere a várias mensagens, não apenas uma. Se estivesse a falar de uma única mensagem, diríamos "mensagem de boas-vindas". Portanto, "mensagens de boas-vindas" indica múltiplas saudações.

Not exactly what you thought. You can only set a message, and not tons of them. So, "mensagem" is already correct.

Reply:
Irei corrigir!